src/lib/Structures/client.ts

```diff
@@ -26,15 +26,15 @@ export default class Client extends SapphireClient {
 		/**
      * Add statuses
      */
-		void this.statusUpdater.addStatus({
+		this.statusUpdater.addStatus({
 			type: 'LISTENING',
 			name: `Signals from the endless outer space | ${cfg.prefix}`
```

Comment:
you could use the internal var parser, please see https://github.com/TMWare/status-rotate#using-variables